javascript/code examples/nodejs.md

```diff
@@ -0,0 +1,18 @@
+## Browser
+Pushing file to lighthouse node using nodejs.
+
+```javascript
+// Deploy file
+const lighthouse = require('lighthouse-web3');
+
```

Comment:
Use self-explanatory words as names. Also comment before each function describing input and return parameters.